Cython/Build/Dependencies.py

```diff
@@ -1023,6 +1027,26 @@ def copy_to_build_dir(filepath, root=os.getcwd()):
                     dir = os.path.dirname(c_file)
                     safe_makedirs_once(dir)
 
+                # write out the depfile, if requested
+                if depfile:
+                    dependencies = deps.all_dependencies(source)
+                    src_base_dir, _ = os.path.split(source)
+                    if not src_base_dir.endswith(os.sep):
+                        src_base_dir += os.sep
+                    # paths below the base_dir are relative, otherwise absolute
+                    paths = []
+                    for fname in dependencies:
+                        if (fname.startswith(src_base_dir) or 
```

Comment:
The code-style failure looks to be trailing white-space on this line. It seems to be passing on the last commit to 0.29.x (https://github.com/cython/cython/runs/4831647955?check_suite_focus=true)

Reply:
Thanks! Fixed